src/NLog/LogFactory.cs

```diff
@@ -804,11 +801,16 @@ protected virtual void OnConfigurationReloaded(LoggingConfigurationReloadedEvent
             ConfigurationReloaded?.Invoke(this, e);
         }
 
-        internal void NotifyConfigurationReloaded(LoggingConfigurationReloadedEventArgs eventArgs)
+        /// <summary>
+        /// Notify success or failed, depending of <paramref name="exception"/>
+        /// </summary>
+        /// <param name="exception">success iif this parameter null</param>
+        internal void NotifyConfigurationReloaded(Exception exception = null)
```

Comment:
FYI: This is a small refactor, as both parameters where highly related, I made it to one.